test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -68,39 +78,88 @@ def test_delete_schedulers(self):
         """
         Delete a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        rc = self.server.manager(MGR_CMD_DELETE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="TestCommonSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
```

Comment:
This try/except block is unnecessary.  The manager() call will raise an exception and fail the test on its own.